.github/workflows/buildAndPush.yml

```diff
@@ -0,0 +1,44 @@
+name: BuildAndPush
+on:
+  pull_request:
```

Comment:
Does this mean that `build` and `publish` will be triggered when PR to the master branch? If so, this should be removed, PRs should not been builded or published until they are merged.